react-backend/routes/notificationTriggers.js

```diff
@@ -0,0 +1,109 @@
+var express = require("express");
+var router = express.Router();
+var mysql = require("mysql");
+var config = require("../config");
+var bodyParser = require("body-parser");
+
+const connection = mysql.createPool({
+  host: config.host,
+  user: config.user,
+  password: config.password,
+  database: config.database
+});
+
+router.post("/", function(req, res, next) {
+  var associatedAccount = req.query.account;
+  var type = req.query.type;
+  var amount = req.query.amount;
+  var value = req.query.value;
+  var startDate = req.query.startDate;
+  var description = req.query.description;
+
+  // convert undefined to null
+
+  if (amount == undefined) {
```

Comment:
This might need to eventually be "undefined" as a string. I know with forms it passes an empty field as a string. Probably fine to leave it as is for now though.